client/javascripts/sagas/auth.js

```diff
@@ -0,0 +1,19 @@
+import { call, takeEvery } from 'redux-saga/effects';
+import * as api from 'api';
```

Comment:
__[eslint]__
* Error - 'api' should be listed in the project's dependencies. Run 'npm i -S api' to add it (import/no-extraneous-dependencies)
* Error - Unable to resolve path to module 'api'. (import/no-unresolved)
* Error - Missing file extension for "api" (import/extensions)

<!-- Comment by [SideCI](https://sideci.com) -->